config/config.go

```diff
@@ -430,6 +413,16 @@ type PrivValidatorConfig struct {
 
 	// Path Root Certificate Authority used to sign both client and server certificates
 	RootCA string `mapstructure:"root-ca-file"`
+
+	// RPC port for Tendermint to query for
+	// an external PrivValidator process
+	PrivValidatorCoreRPCHost string `mapstructure:"core-rpc-host"`
```

Comment:
wdyt to shorten the name of this field and others that begins with `PrivValidator` by removing this prefix since they are already placed in the `PrivValidatorConfig` structure?